js/index.js

```diff
@@ -0,0 +1,104 @@
+// eslint-disable-next-line no-unused-vars
+function openNav() {
+  document.getElementById('myNav').style.width = '100%';
+}
+
+// eslint-disable-next-line no-unused-vars
+function closeNav() {
+  document.getElementById('myNav').style.width = '0%';
+}
+// my product section
+const project = [
+  {
+    id: 0,
+    img: './Images/Pink-gel.jpeg',
+    title: 'Pink-Gel',
+    supportingText: 'A color builder gel which can be mix with other colorful gel to have your desirable color',
+    button: 'Buy Now',
+  },
+  {
+    id: 1,
+    img: './Images/Nail-Set.jpeg',
+    title: 'Nail Set',
+    supportingText: 'Main Equipements every nail tech needs to get into the nail business althought not a complete set',
+    button: 'Buy Now',
+
+  },
+  {
+    id: 3,
+    img: './Images/Foot-Bath-Fizz.jpeg',
+    title: 'Foot Bath Fizz',
+    supportingText: 'It helps refresh and revive hard working feet. softening and refreshing the fizzy foot bath contains cooling peppermint and antibacterial Tea tree essential oil',
+    button: 'Buy Now',
+
+  },
+  {
+    id: 4,
+    img: './Images/Gel-Scrub.jpeg',
+    title: 'Gel Scrub',
+    supportingText: 'This soothes and eliminates buildup of dead skin, moisturizes and nourishes while regenerating the skin back to a healthy state',
+    button: 'Buy Now',
+
+  },
```

Comment:
- Note that for better UI, your design has to match that of the original template
Your design | Behance design
![screenshot (6)](https://user-images.githubusercontent.com/44624138/181983965-e42a8a76-d8df-4bdf-a223-826244ba32b0.png) | ![screenshot (5)](https://user-images.githubusercontent.com/44624138/181984408-27ed94aa-b600-4d55-b89e-0faaffe2c739.png)